aiohttp/http_writer.py

```diff
@@ -114,6 +114,7 @@ async def write(
                 chunk = chunk_len_pre + chunk + b"\r\n"
 
             self._write(chunk)
+            self.protocol.data_received(b"")
```

Comment:
How does this correspond to https://github.com/aio-libs/aiohttp/blob/master/aiohttp/_http_writer.pyx?

Reply:
Seems like we're going to make the changes elsewhere, so this should be irrelevant once updated.